package 아무코드.exception;

public class Main {

    public static void main(String[] args) {


        int result1 = 30 / 0;
        
//        try {
//            int result = 30 / 0; // Trying to divide by zero
//        } catch (ArithmeticException e) {
//            System.out.println("ArithmeticException caught!");
//        }

    }


}
